tests/workflows/test_ao_ic_usd.py

```diff
@@ -0,0 +1,58 @@
+import os
+from argparse import Namespace
+
+import pandas as pd
+
+from mawpy.constants import (USER_ID, UNIX_START_T, UNIX_START_DATE, ORIG_LAT, ORIG_LONG, ORIG_UNC, STAY_LAT, STAY_LONG,
+                             STAY_UNC, STAY, STAY_DUR)
+from mawpy.steps.address_oscillation import address_oscillation
+from mawpy.steps.incremental_clustering import incremental_clustering
+from mawpy.steps.update_stay_duration import update_stay_duration
+from mawpy.workflows.ao_ic_usd import ao_ic_usd
+
+
+def test_workflow_ao_ic_usd(tmp_path):
+
+    input_file = os.path.dirname(__file__) + '/../resources/test_input.csv'
+    output_file = os.path.join(tmp_path, 'test_output_workflow_ao_ic_usd.csv')
+    # Prepare arguments similar to command-line arguments
+    args = Namespace(
```

Comment:
why put these values in a Namespace?

Reply:
Using Namespace to make the invocation call to the workflow similar to how it's done in the ao_ic_usd.py with the command line arguments accessed by args.<param_name>